.travis.yml

```diff
@@ -99,8 +99,8 @@ install:
   - "mkdir -p $TRAVIS_BUILD_DIR/build/release"
   # Enable test coverage for travis-ci build
   - "cmake $TRAVIS_BUILD_DIR -DNTA_COV_ENABLED=ON -DCMAKE_INSTALL_PREFIX=${TRAVIS_BUILD_DIR}/build/release -DPY_EXTENSIONS_DIR=$TRAVIS_BUILD_DIR/bindings/py/nupic/bindings"
-  - "VERBOSE=1 make"
-  - "make install"
+  - "VERBOSE=1 make|grep -v -F '\\-\\- Installing:'"
+  - "make install 2>&1|grep -v -F 'Installing:'"
```

Comment:
I thought something like errexit was already the default behavior??? If any of the command lines fails, doesn't the build automatically fail at that point? If not, then errexit would be a good thing.